config/initializers/split.rb

```diff
@@ -0,0 +1,5 @@
+Split.configure do |config|
+  config.db_failover = true
+  config.enabled = Rails.env.test? ? false : true
```

Comment:
Perhaps later we might want more control over this, maybe via a setting in application.yml so that we can turn it on and off in production.